### ISSUE TITLE
[mdc-typography] Unable to overwrite styles in v5

### ISSUE THREAD

**Issue by kimurakenshi** [material-components/material-components-web#5648]
## Bug report

I'm migrating to v5 and while trying to override headline styles using the sass modules syntax an error is displayed during the compilation phase.

### Steps to reproduce

1. Compile the following scss with sass:

```scss
@use "@material/typography" with (
 $styles-headline1: (
    font-weight: 300,
  ),
);
```

### Actual behavior

```bash
formatted: `Error: This module was already loaded, so it can't be configured using "with".\n` +
    '   ┌──> node_modules/@material/typography/_index.scss\n' +
    '3  │   @forward "./functions";\n' +
    '   │   ^^^^^^^^^^^^^^^^^^^^^^ new load\n' +
    '   ╵\n' +
    '   ┌──> node_modules/@material/typography/_variables.scss\n' +
    '25 │   @use "./functions";\n' +
    '   │   ━━━━━━━━━━━━━━━━━━ original load\n' +
```

### Expected behavior

Styles should be overwritten.

### Additional context

I'm able to overwrite the font-family without issues when i compile:

```scss
@use "@material/typography" with (
 $font-family: unquote("MyCustomFont, Verdana, Arial, sans-serif")
);
```
I'm following the steps described in https://github.com/material-components/material-components-web/tree/master/packages/mdc-typography#overriding-styles

**Comment by asyncLiz** [comment]
Duplicate of #5621. This issue was fixed today but has not been released yet. You can verify the fix by installing the canary version of `@material/typography`.

I'll check in tomorrow to see if there's a patch release on the horizon that we can cut.

**Comment by kimurakenshi** [comment]
Thank you @asyncLiz, i appreciate you quick reply on this.

> I'll check in tomorrow to see if there's a patch release on the horizon that we can cut.

That will be great! So i will be able to continue the migration to v5.

**Comment by asyncLiz** [comment]
Should be published now in [v5.1.0](https://github.com/material-components/material-components-web/releases/tag/v5.1.0).
